beesdoo_pos/__manifest__.py

```diff
@@ -9,15 +9,17 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "BEES coop Point of sale",
-    "summary": """This module adds the eaters of the customer to the POS
-    ActionpadWidget and PaymentScreenWidget.""",
+    "summary": """Emptied.""",
     "author": "BEES coop - Cellule IT, Coop IT Easy SC",
     "website": "https://github.com/beescoop/Obeesdoo",
     "category": "Point Of Sale",
     "version": "12.0.2.0.0",
-    "depends": ["eater", "beesdoo_product"],
-    "data": ["views/beesdoo_pos.xml", "data/default_barcode_pattern.xml"],
-    "qweb": ["static/src/xml/templates.xml"],
+    "depends": [
+        "pos_eater",
+        "pos_available_in_pos_unlink",
+        "pos_forbid_new_customer",
```

Comment:
@victor-champonnois is this another module than `pos_partner_no_create`?

Reply:
No it's the same